lib/Doctrine/DBAL/Driver/PDOPgSql/Driver.php

```diff
@@ -88,11 +88,8 @@ private function _constructPdoDsn(array $params)
 
         if (isset($params['dbname'])) {
             $dsn .= 'dbname=' . $params['dbname'] . ' ';
-        } else {
-            // Used for temporary connections to allow operations like dropping the database currently connected to.
-            // Connecting without an explicit database does not work, therefore "postgres" database is used
-            // as it is certainly present in every server setup. 
-            $dsn .= 'dbname=postgres' . ' ';
+        } elseif (isset($params['default_dbname'])) {
+            $dsn .= 'dbname=' . $params['default_dbname'] . ' ';
```

Comment:
@kimhemsoe that is not true. It defaults to a database that is named equally to the connection user. This is what originally caused trouble. See [PHP issue](https://bugs.php.net/bug.php?id=67431) and https://github.com/doctrine/DoctrineBundle/issues/402